curvine-server/src/worker/storage/vfs_dataset.rs

```diff
@@ -28,6 +28,7 @@ pub struct VfsDataset {
     worker_id: u32,
     ctime: u64,
     dir_list: DirList,
+    dir_block_nums: HashMap<u32, i64>,
```

Comment:
In fact, we can get the total number of blocks by directly counting the number of block_maps. dir_block_nums should not be needed, and this data is not updated when adding or deleting blocks.